crates/phaxt/src/lib.rs

```diff
@@ -1,14 +1,16 @@
 use parity_scale_codec::{Decode, Encode};
 use scale_info::TypeInfo;
 
-#[subxt::subxt(runtime_metadata_path = "metadata_files/khala_metadata.scale")]
+// #[subxt::subxt(runtime_metadata_path = "metadata_files/khala_metadata.scale")]
+#[subxt::subxt(runtime_metadata_path = "metadata_files/phala_standalone_metadata.scale")]
```

Comment:
Is it for local dev? no need to commit.

Reply:
Yes, for temporary usage (because I changed extrinsics and imcompatible with current)

Reply:
This must be commit, or the subxt will use imcompatible metadata, which break pherry submit transactions